CMakeLists.txt

```diff
@@ -465,49 +465,23 @@ SET(BUILD_SHARED_LIBS ON)
 ## HEADER/LIBRARY/OTHER CHECKS ##
 # First, required stuff
 if(USE_LIBSNDFILE)
- set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUSE_LIBSNDFILE")
- set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_LIBSNDFILE")
- if(MSVC)
-    find_package(SndFile CONFIG REQUIRED)
- else()
-    find_library(LIBSNDFILE_LIBRARY NAMES sndfile libsndfile-1 libsndfile)
-
-  if(NOT LIBSNDFILE_LIBRARY)
-        message(FATAL_ERROR "Csound requires the sndfile library")
-   endif()
- endif()
-endif()
-
-if(EMSCRIPTEN)
-  include_directories(emscripten/deps/libsndfile-1.0.25/src/)
-endif()
-find_path(SNDFILE_H_PATH sndfile.h)
-if(SNDFILE_H_PATH)
-    include_directories(${SNDFILE_H_PATH})
-else()
-    message(FATAL_ERROR "Could not find sndfile.h")
-endif()
-
-SET(CMAKE_REQUIRED_LIBRARIES ${LIBSNDFILE_LIBRARY})
-SET(CMAKE_REQUIRED_INCLUDES ${SNDFILE_H_PATH})
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUSE_LIBSNDFILE")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_LIBSNDFILE")
 
-check_c_source_compiles("#include <sndfile.h>
-int main(int argc, char** argv) {
-printf(\"SF_FORMAT_MPEG: %d\", SF_FORMAT_MPEG);
-return 0;
-}" HAVE_SF_FORMAT_MPEG)
```

Comment:
Not sure this should be deleted. This did actually catch an issue recently. It is possible to have libsndfile without this sub dependency but the code will break without that. This ensures it's obvious to the user why.